docs/workflows-config.md

```diff
@@ -81,6 +134,10 @@ A named group of Bazel commands that run when triggered.
 - **`name`** (`string`): A name unique to this config, which shows up as the name of the check
   in GitHub.
 - **`triggers`** ([`Triggers`](#triggers)): The triggers that should cause this action to be run.
+- **`os`** (`string`): The operating system on which to run the workflow.
```

Comment:
if you add arch here too then i think we'll be good

Reply:
Done